vignettes/bds_finding.Rmd

```diff
@@ -626,92 +781,38 @@ advs <- advs %>%
   ) %>%
   union_all(advs) %>%
   select(-EOTFL)
+```
 
-advs %>% 
-  select(USUBJID, PARAMCD, ADT, AVISITN, AVISIT, ATPTN, AVAL, ANL01FL) %>% 
-  filter(ANL01FL == "Y") %>% 
-  arrange(USUBJID, PARAMCD, ADT, AVISITN, ATPTN) %>% 
-  head(n = 10)
+```{r, eval=TRUE, echo=FALSE}
+dataset_vignette(advs_ex1, 
+                 display_vars = vars(USUBJID, PARAMCD, ADT, AVISITN, AVISIT, ATPTN, AVAL, ANL01FL),
+                 filter = USUBJID == "01-701-1015" & ANL01FL == "Y"
+)
 ```
 
+
 Example 2 (Deriving a Summary record):
 
 ```{r eval=TRUE}
-advs <- derive_summary_records(
+advs_ex2 <- derive_summary_records(
   advs,
   by_vars = vars(STUDYID, USUBJID, PARAMCD, VISITNUM, ADT),
   analysis_var = AVAL,
   summary_fun = mean,
   set_values_to = vars(DTYPE = "AVERAGE")
 )
-
-head(select(advs, USUBJID, PARAMCD, VISITNUM, ADT, AVAL, DTYPE))
-```
-
-Example 3 (Derive a new PARAMCD):
```

Comment:
I think it would be good to show an example on how to use the derive_derived_param function. Even if not explicitely called: it may be good to show an example: Maybe with another MAP formula?
```
derive_derived_param(
   advs,
   by_vars = vars(USUBJID, VISIT),
   parameters = c("SYSBP", "DIABP"),
   analysis_value = (AVAL.SYSBP-AVAL.DIABP)/3 + AVAL.DIABP ,
   set_values_to = vars(
     PARAMCD = "MAP2",
     PARAM = "Mean Arterial Pressure 2 (mmHg)"
   )
 )

```

Reply:
Done